src/tests/ftest/launch.py

```diff
@@ -711,6 +712,20 @@ def replace_yaml_file(yaml_file, args, tmp_dir):
     # Return the untouched or modified yaml file
     return yaml_file
 
+def generate_certs(args):
+    """Generate the certificates for the test."""
+    subprocess.call(
+        ["/usr/bin/rm", "-rf",
+         "{}/certs".format(os.environ["DAOS_TEST_LOG_DIR"])])
+    subprocess.call(["../../../../lib64/daos/certgen/gen_certificates.sh"])
+    subprocess.call(
+        ["/usr/bin/mv", "daosCA/certs",
+         "{}/".format(os.environ["DAOS_TEST_LOG_DIR"])])
+    spawn_commands(args.test_servers.split(","), "ls -la {}".format(
+        os.path.join(os.environ["DAOS_TEST_LOG_DIR"], "certs")))
+    subprocess.call(
+        ["/usr/bin/ls", "-la", os.path.join(
+            os.environ["DAOS_TEST_LOG_DIR"], "certs")])
```

Comment:
I do not believe the `gen_certificates.sh` file will have permissions to write into the daos_test RPM `ftest` directory.   The ideal solution is to for `gen_certificates.sh` to support a `<destination_directory>` argument (in place of assuming `.`) and then this code could be simplified down to:
```suggestion
    certs_dir = os.path.join(os.environ["DAOS_TEST_LOG_DIR"], "certs")
    subprocess.call(["/usr/bin/rm", "-rf", certs_dir])
    subprocess.call(["../../../../lib64/daos/certgen/gen_certificates.sh", certs_dir])
```